src/Simplex/Chat/Library/Internal.hs

```diff
@@ -2008,7 +2008,7 @@ saveSndChatItems ::
 saveSndChatItems user cd itemsData itemTimed live = do
   createdAt <- liftIO getCurrentTime
   when (contactChatDeleted cd || any (\NewSndChatItemData {content} -> ciRequiresAttention content) (rights itemsData)) $
-    withStore' (\db -> updateChatTs db user cd createdAt)
+    withStore' (\db -> updateChatTsStats db user cd createdAt Nothing)
```

Comment:
when item is sent, if the member is not the user, unanswered count should be reset to 0 (others won't change).